contributing/contributing_to_datacommons.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Contributing to Data Commons
+nav_order: 6
+has_children: true
+---
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+## Contributor License Agreement
+
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see
+your current agreements on file or to sign a new one.
+
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
+
+## Code reviews
+
+All submissions, including submissions by project members, require review. We
+use GitHub pull requests for this purpose. Consult
+[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
+information on using pull requests.
+
```

Comment:
Could you add this section here:

## Style Guide

All code contributions should follow Google language-specific style guides at
<https://google.github.io/styleguide>.

Reply:
Should the style guide link be https://google.github.io/styleguide/ or https://developers.google.com/style? I've been using the second as a reference, but if you'd rather use the more OSS-specific one, that's perfectly alright!